bouncer/shared/fund_redeem.ts

```diff
@@ -29,7 +29,7 @@ async function redeemAndObserve(
   await redeemFlip(seed, redeemEthAddress, redeemAmount);
 
   const newBalance = await observeBalanceIncrease('FLIP', redeemEthAddress, initBalance);
-  const balanceIncrease = newBalance - parseInt(initBalance);
+  const balanceIncrease = newBalance - parseFloat(initBalance);
```

Comment:
FYI: Fixed a bug here.

Reply:
I don't know if this will completely fix it, I also added an amount of precision, here:  https://github.com/chainflip-io/chainflip-backend/pull/4274/commits/364b3c6df05f24c6b7ba6d64ee07fc1077bc0449